manifest_staging/charts/gatekeeper/templates/gatekeeper-mutating-webhook-configuration-mutatingwebhookconfiguration.yaml

```diff
@@ -2,7 +2,6 @@
 apiVersion: admissionregistration.k8s.io/v1
 kind: MutatingWebhookConfiguration
 metadata:
-  annotations: {{- toYaml .Values.mutatingWebhookAnnotations | trim | nindent 4 }}
```

Comment:
why are these being removed?

Reply:
run make manifests to automatically delete

Reply:
I don't think we would want these to be deleted, we might want to check why new kustomize is deleting them

Reply:
t is indeed a problem caused by the upgraded version of kustomize. I first submitted the pr of yq. I solved the problem of kustomize and submitted it with another branch. This is the pr of yq https://github.com/open-policy-agent/gatekeeper/pull/2497